app/scripts/icons.js

```diff
@@ -369,3 +371,18 @@ export const svgArrowheadDomainsIcon = parser.parseFromString(
   arrowHeadDomainsStr,
   'text/xml',
 ).documentElement;
+
+export function SearchIcon({ theStyle }) {
+  return (
+    <svg
+      styleName={theStyle}
```

Comment:
What is the `styleName` property used for? Should this be `className` or `style` or is this a special property?

Reply:
It's how we pass style names using `react-css-modules`. Tbh, it feels a bit dated but that's what we use in other places as well so I'm not too keen on changing it just yet.